requirements.txt

```diff
@@ -19,9 +19,11 @@ pycparser==2.20
 pydantic==1.7.3
 PyJWT==2.0.1
 python-dateutil==2.8.1
-pytz==2021.1
+pytz==2020.4
 PyYAML==5.4.1
+raven==6.10.0
 requests==2.25.1
+requests-futures==1.0.0  # Pending 3PP
```

Comment:
What's the 3pp status?

Reply:
https://gus.lightning.force.com/lightning/r/0D5B000001DKiT7KAL/view
Your third-party component review request has been approved. Good work, and thank you.